packages/terra-date-time-picker/translations/de.json

```diff
@@ -1,6 +1,6 @@
-{
-  "Terra.dateTimePicker.timeClarification.title": "Klärung der Zeit",
-  "Terra.dateTimePicker.timeClarification.message": "Die ausgewählte Zeit liegt während der Umstellung von Sommer- auf Standardzeit. Möchten Sie für die Zeit vor oder nach der Zeitumstellung dokumentieren?",
-  "Terra.dateTimePicker.timeClarification.button.daylightSaving": "Vor (Sommerzeit)",
-  "Terra.dateTimePicker.timeClarification.button.standardTime": "Nach (Standardzeit)"
+{
```

Comment:
The only thing that changed in these translations was end of line encodings. No idea why my computer does this sometimes...